tests/components/climate/test_reproduce_state.py

```diff
@@ -48,21 +46,22 @@
     calls_1 = async_mock_service(hass, DOMAIN, SERVICE_SET_HVAC_MODE)
 
     await async_reproduce_states(
-        hass, [State(ENTITY_1, HVAC_MODE_HEAT), State(ENTITY_2, HVAC_MODE_AUTO)]
-    )
+        hass,
+        [State(ENTITY_1, HVAC_MODE_HEAT),
+         State(ENTITY_2, HVAC_MODE_AUTO)])
 
     await hass.async_block_till_done()
 
     assert len(calls_1) == 2
     # order is not guaranteed
-    assert any(
-        call.data == {"entity_id": ENTITY_1, "hvac_mode": HVAC_MODE_HEAT}
-        for call in calls_1
-    )
-    assert any(
-        call.data == {"entity_id": ENTITY_2, "hvac_mode": HVAC_MODE_AUTO}
-        for call in calls_1
-    )
+    assert any(call.data == {
+        "entity_id": ENTITY_1,
+        "hvac_mode": HVAC_MODE_HEAT
+    } for call in calls_1)
+    assert any(call.data == {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4636028)